chain/src/tests/basic.rs

```diff
@@ -73,7 +73,8 @@ fn test_genesis_transaction_spend() {
 
 #[test]
 fn test_transaction_spend_in_same_block() {
-    let (chain_controller, shared) = start_chain(None, true);
+    let consensus = Consensus::default().set_cellbase_maturity(0);
```

Comment:
I perfer setting default test `cellbase_maturity` value in `start_chain`  helper.

Reply:
Updated.